packages/orbit-components/src/Itinerary/ItinerarySegment/context.jsx

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as React from "react";
+
+import type { Context } from "./context";
+import typeof { ItinerarySegmentProvider as ItinerarySegmentProviderType } from "./context";
+
+export const ItinerarySegmentContext: React.Context<Context> = React.createContext({
+  isNextHidden: false,
+  isHidden: false,
+  noElevation: false,
+  opened: false,
+  toggleOpened: () => {},
+  index: 0,
+  count: 0,
+  last: false,
+});
+
+export const ItinerarySegmentProvider: ItinerarySegmentProviderType = ({
+  children,
+  ...ctx
+}): React.Node => {
```

Comment:
No need for return type because it's already a part of `ItinerarySegmentProviderType`.
```suggestion
}) => {
```